akka-stream/src/main/scala/akka/stream/QueueOfferResult.scala

```diff
@@ -5,9 +5,10 @@
 package akka.stream
 
 sealed abstract class QueueOfferResult
+sealed abstract class QueueCompletionResult extends QueueOfferResult
```

Comment:
Add some `@DoNotInherit` for these two as well, sealed doesn't help Java users.

Reply:
:+1: